### PR TITLE
handle top level consts as doc references

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Addresses https://github.com/dart-lang/dartdoc/issues/761

Also, WIP

**Comment by sethladd** [comment]
Blocked by https://github.com/dart-lang/markdown/issues/42

**Comment by sethladd** [comment]
@devoncarew please take a look. Thanks!

File: lib/src/debug.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+library debugger_helper;
+
+import "dart:developer" as dev;
+
+get debugger =>
```
Comment (devoncarew) [inline lib/src/debug.dart]: cool

File: lib/src/model.dart [hunk 2]
```diff
@@ -14,6 +14,7 @@ import 'package:quiver/core.dart';
 import 'html_utils.dart';
 import 'model_utils.dart';
 import 'package_meta.dart';
+import 'debug.dart';
```
Comment (devoncarew) [inline lib/src/model.dart]: nit: sorting

**Comment by devoncarew** [comment]
lgtm

re: the dartfmt output, you can always disable the formatting check for now. That way we'll still have analysis and test coverage.

**Pull request closed without merging**